Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,68 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+/// A gaussian noise layer
+///
+/// Gaussian Noise adds noise sampled from a normal distribution with mean zero
+public struct GaussianNoise<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  @noDerivative public let standardDeviation: Tensor<Scalar>
+  
+  /// Creates a Gaussian noise layer
+  ///
+  /// - Parameter standardDeviation: Standard deviation of the Guassian distribution
+  public init(standardDeviation: Scalar) {
```

Comment:
Could we have an overloaded `init` that takes `standardDeviation` as a `Tensor<Scalar>` in addition to `Scalar`? (Note: I think it's great to have this one, so please don't get rid of this one.)

Reply:
Due to changing `standardDeviation` to a `Scalar` (instead of `Tensor<Scalar>`), let's ignore this comment now. Thanks!